drivers/ieee802154/ieee802154_kw41z.c

```diff
@@ -530,6 +530,11 @@ static inline void kw41z_rx(struct kw41z_context *kw41z, u8_t pkt_len)
 	net_pkt_set_ieee802154_lqi(pkt, kw41z_convert_lqi(hw_lqi));
 	/* ToDo: get the rssi as well and use net_pkt_set_ieee802154_rssi() */
 
+	if (ieee802154_radio_handle_ack(kw41z->iface, pkt) == NET_OK) {
+		SYS_LOG_DBG("ACK packet handled");
```

Comment:
Just FYI but the HW is configured to handle the ACKs so they won't be passed up to the driver...

Reply:
oh yes, completely forgot about IEEE802154_HW_TX_RX_ACK capability. Vacations were nice.